app/models/profile.rb

```diff
@@ -13,13 +13,12 @@ def contact(other_profile)
 
     if !other_profile.driver?
       msg = %[
-        RideWithBernie: #{first_name} at #{phone} offered to drive you to #{event_title}, 
-        coming from #{location} with #{seats} seat#{'s' if seats != 1} available.
+        RideWithBernie: #{first_name} at #{phone} from #{location} offered to drive you to #{event_title}. Can you go?
```

Comment:
Nvm! Didn't see this convo was already happening in #17. I'll keep quiet until that is resolved there.